src/components/molecules/TabsContainer.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import TabMenu from './TabMenu';
+
+interface IProps {
+  className?: string;
+  children: Array<JSX.Element>;
+}
+
+interface IPropsTabContent {
+  className?: string;
+  id: string;
+  labelledBy: string;
+  role: string;
+  label: string;
+  activeTab: string;
+}
+
+const TabContent: React.FC<IPropsTabContent> = props => {
+  return (
+    <div
+      className={props.className}
+      id={props.id}
+      role={props.role}
+      aria-labelledby={props.labelledBy}
+    >
+      {props.children}
+    </div>
+  );
+};
+
+const TabsContainer: React.FC<IProps> = props => {
+  const [activeTab, setActiveTab] = React.useState(
+    props.children[0].props.label
+  );
+
+  const tabLabels = new Array(props.children.length);
+  props.children!.forEach(child => {
+    tabLabels.push(child.props.label);
+  });
+
+  return (
+    <div className={props.className}>
+      <TabMenu
+        className="tab-menu"
+        tabLabels={tabLabels}
+        activeTab={activeTab}
+        setActiveTab={setActiveTab}
+      />
+      {props.children
+        .filter(child => child.props.label === activeTab)
+        .map(child => (
```

Comment:
As you are only using `child.props.label` and `child.props.children`, so if you wish, you could deconstruct the incoming argument as following;

```typescript
.map(({ props: { label, children } }) => 
```

This would make the body of the anonymous function shorter, but would perhaps compromise the legibility of the argument. Personally I find that a worthwhile tradeoff, but that's personal preference.

Reply:
That would make e.g. this line

```typescript
id={`${child.props.label}-tab`}
```

into this

```typescript
id={`${label}-tab`}
```

Reply:
Good solution :+1: I put the destructuring together with the find-method and moved it outside the return expression:
```const {
    props: { label, children },
  } = props.children.find(child => child.props.label === activeTab)!;
```